2022/questions.md

```diff
@@ -0,0 +1,531 @@
+* 1 – Extremely critical
+* 2 – Very critical
+* 3 – Somewhat critical
+* 4 – Not so critical
+* 5 – Not at all critical
+
+19. Does your organization use its open source program office as a way to further its strategic relationships and build partnerships?
+* Yes
+* No
+* Don't know
+
+20. Has the open source program had a positive impact on your company's software practices?
+* Yes
+* No
+* Don't know
+
+>ROUTING: "Yes" goes to Part 2d; "No" and "Don't know" go to Part 5
+### Part 2d: Have Program - yes, specific impact
+21. If yes, please provide one or two specific examples.
+* Example 1
+* Example 2
+
+>Routing: go to Part 5a
+## Part 3: Planning an Open Source Program
+22. When does your company plan to start a program?
+* In the next 6 months
+* In the next year
+* 1-2 years from now
+* Over 2 years from now
+
+23. Will the program or initiative be formally structured with dedicated person-hours, reporting structure and/or job titles?
+* Yes
+* No
+* Don't know
+
+24. Where will the open source program or initiative be located within the organization? If the effort is informal, answer based on who the primary organizers will report to.
+* Legal
+* Software engineering and development
+* IT
+* Office of the CTO
+* Developer relations, marketing or communications
+* Security, compliance or risk management
+* Don't know
+* Other (please specify)
+<!---changed to "or risk management" --->
+
+25. What does your company aim to accomplish by starting an open source program? (Check all that apply)
+* Increased developer recruitment and retention
+* Increased speed and agility in development cycle
+* Better license compliance
+* Lower licensing fees
+* Lower support costs
+* More influence in open source communities
+* Increased contributions to in-house open source projects from external or third-party contributors
+* More awareness of open source use and commercial dependencies
+* Increased market adoption of open source projects
+* Increased participation in external open source projects
+* Faster time to market with new products
+* Increased innovation
+* Culture change, improving interaction among departments
+* Better security testing and vulnerability management
+* Other (please specify)
+
+26. What have been the top three biggest challenges in establishing an open source program? (Choose three)
+* Developing an open source strategy
+* Finding an open source program manager
+* Finding legal staff with open source expertise
+* Setting an open source policy
+* Getting executive support and buy-in
+* Getting engineering support and buy-in
+* Setting a budget and estimating program costs
+* Assessing or quantifying existing open source use and contribution
+* Resources required to perform license compliance
+* Finding commercial dependencies
+* Tool selection
+* Other (please specify)
+>Routing: go to Part 5a
+## Part 4
+### Part 4a: Do Not Have an Open Source Program
```

Comment:
```suggestion
## Part 4

### Part 4a: Do Not Have an Open Source Program

```

Similar - whenever there's headers, you want a full line between them or else markdown renderers sometimes get confused